core/src/main/resources/lib/form/textarea.jelly

```diff
@@ -89,6 +89,7 @@ THE SOFTWARE.
             readonly="${attrs.readonly}"
             codemirror-mode="${attrs['codemirror-mode']}"
             codemirror-config="${attrs['codemirror-config']}">
+    <j:if test="${value != null &amp;&amp; !empty(value.toString()) &amp;&amp; (value.toString().codePointAt(0) == 10 || value.toString().codePointAt(0) == 13)}"><j:whitespace>&#10;</j:whitespace></j:if>
```

Comment:
I tried this in jelly
```
:\r\n
:${'\r\n'}
:${&quot;\r\n&quot;}
:<st:out value="\r\n" />
:<st:out value="${'\r\n'}" />
:<st:out value="${&quot;\r\n&quot;}" />
```
and that resulted like this:
```
:\r\n
:\r\n
:\r\n
:\r\n
:\r\n
:\r\n
```
backslashes doesn't seem handled.

Maybe I need another way to handle control characters.
Would you show me some examples to have jelly unescape texts?
I could not find good documentations for jelly...